components/usergroup/joinGithub.tsx

```diff
@@ -0,0 +1,47 @@
+import Image from "next/image";
+import { Template } from "tinacms";
+import { tinaField } from "tinacms/dist/react";
+
+export const JoinGithub = ({ data }) => {
+  return (
+    <div className="flex flex-col items-center justify-center gap-y-3 rounded-xl bg-gray-900 bg-arcBackground bg-bottom bg-no-repeat px-11 py-5 text-center font-helvetica">
+      <Image
+        src={"/images/badges/Github-Badge.svg"}
+        alt="github image"
+        width={80}
+        height={80}
+        className="mx-auto"
+      />
+      <div
+        className="text-center text-md text-white"
+        data-tina-field={tinaField(data, "title")}
+      >
+        {data?.title}
+      </div>
+      <button
+        data-tina-field={tinaField(data, "link")}
+        onClick={() => window.open(data?.link ?? "", "_blank").focus}
+        className="h-12 w-max rounded-md bg-sswRed px-4 py-2 text-md leading-3 text-white"
+      >
+        Join the crowd
+      </button>

```

Comment:
Could this be an <a> tag?

Reply:
I'm not fully understand this.
What does the `tag` referring to? I may reach you tomorrow for this.